### ISSUE TITLE
Allow collaborative editing for shared annotations

### ISSUE THREAD

**Issue by fm3** [scalableminds/webknossos#5789]
Shared annotations are view-only for all who are not the original owner.
It would be cool to enable saving in annotations that were explicitly shared with me.

Users will have to take care not to create version conflicts.
This can be supported by
 - starting saving only after a „meaningful“ change was made (moving the camera does not count)
 - (alternatively an explicit “I want to edit now” interaction)
Update Actions need an author field (is some validation needed in tracingstore?)

**Comment by fm3** [comment]
superseded by #6227 and closed in #6236
